### PR TITLE
150 tweaks to pkgdown page

### PR DESCRIPTION
**Pull request by MikeJSeo**
okay I started this..

### CODE REVIEW

**Comment by gravesti** [comment]
Thanks @MikeJSeo

**Comment by MikeJSeo** [comment]
Okay, checks all passed! Please modify this as needed. Thanks! @gravesti

**Comment by MikeJSeo** [comment]
Just copying notes for README for ref:

Short intro to package, see DESCRIPTION
links to poster and abstract
how to install dev version, eg with remotes::install_github("hta-pharma/maicplus") (to be updated when on CRAN )
Links to vignettes
Link to NICE TSD showing why we might use MAIC

**Comment by gravesti** [comment]
Thanks @MikeJSeo 
I will try to update the logo files tomorrow, something went awry. Then we can merge

**Review by gravesti — APPROVED** [review]

File: DESCRIPTION [hunk 1]
```diff
@@ -2,8 +2,40 @@ Title: Matching Adjusted Indirect Comparison
...
+      role = c("ctb"),
+      email = "miranta.antoniou@roche.com"
+      ),
+   person(
+      given = "Chrysostomos",
+      family = "Kalyvas",
+      role = c("ctb"),
+      email = "chrysostomos.kal@gmail.com"
+   ),
+   person("MSD, Inc.", role = c("cph", "fnd")),
```
Comment (gravesti) [inline DESCRIPTION:36]:
```suggestion
   person("MSD Merck Sharp & Dohme AG", role = c("cph", "fnd")),
```
@hoppanda maybe like this?

**Pull request merged**